zen/scene/screen-layout.c

```diff
@@ -0,0 +1,48 @@
+#include "zen/scene/screen-layout.h"
+
+#include "zen-common.h"
+#include "zen/scene/screen.h"
+
+void
+zn_screen_layout_add(
+    struct zn_screen_layout* self, struct zn_screen* new_screen)
+{
+  wl_list_insert(&self->screens, &new_screen->link);
+  int x = 0;
+  struct zn_screen* screen;
+  struct wlr_box box;
+  wl_list_for_each(screen, &self->screens, link)
+  {
+    wlr_output_effective_resolution(
+        screen->output->wlr_output, &box.width, &box.height);
+    screen->x = x;
+    screen->y = 0;
+    x += box.width;
+  }
```

Comment:
int x doesn't seem necessary.
I want you to declare variable at the beginning of the scope.
```suggestion
  struct zn_screen* screen;
  struct wlr_box box;
  wl_list_insert(&self->screens, &new_screen->link);
  wl_list_for_each(screen, &self->screens, link)
  {
    wlr_output_effective_resolution(
        screen->output->wlr_output, &box.width, &box.height);
    screen->x += box.width;
    screen->y = 0;
  }
```